### PR TITLE
Fix noticias plus

### PR DESCRIPTION
**Pull request by mikeyrayvon**
add graphic asset. position and size for desktop and mobile

### CODE REVIEW

File: page-home.php [hunk 1]
```diff
@@ -12,8 +12,16 @@
 <!-- end main-content -->
 </main>
 
-<nav id="home-plus">
-  <a href="<?php echo site_url('/noticias'); ?>">+</a>
+<nav class="home-plus only-desktop">
+  <div class="container">
```
Comment (mixcas) [inline page-home.php:16]: why does this need `.container .row. col`?
IMO it should be as simple as `nav a` with some utility classes
Also this markup is overlapping the content in`.main-content`

**Comment by mikeyrayvon** [comment]
grid because look at the design. i fixed the pointer events so the overlap doesnt interupt

File: page-home.php (hunk 1 above)
Comment (mixcas) [reply page-home.php:16]: ok, i see it now

File: css/site.styl [hunk 2]
```diff
@@ -309,6 +309,19 @@ body.single-programacion
...
+.home-plus
+  pointer-events: none
+  a 
+    pointer-events: auto
+    svg
+      height: 50px
+  &.only-desktop
+    position: fixed
+    text-align: right
+    top: 60%
```
Comment (mixcas) [inline css/site.styl]: I think its too close to the center. to me, from the design, it seem to be positioned at ~`bottom: 18%`

**Comment by mixcas** [comment]
is this ready?

**Pull request merged**